test/emoji.js

```diff
@@ -77,4 +77,40 @@ describe("emoji.js", function () {
     should.exist(coffee);
     coffee.should.be.exactly('☕️');
   });
+
+  describe("find(str)", function () {
+    it("should return partially matched emojis", function () {
+      var matchingEmojis = emoji.search("cof");
+      var coffee = matchingEmojis[0];
```

Comment:
I wouldn't rely on coffee being the first result here (in case more emoji get added later), check if one of the matches was coffee instead